omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -383,6 +383,8 @@ def gen_backend_default(bootstrap)
       PrivateChef['bookshelf']['listen'] ||= PrivateChef['default_listen_address']
       PrivateChef['rabbitmq']['node_ip_address'] ||= PrivateChef['default_listen_address']
       PrivateChef['redis_lb']['listen'] ||= PrivateChef['default_listen_address']
+      # Should there be value for es here?
+      #PrivateChef['elasticsearch']['ip_address'] ||= PrivateChef['default_listen_address']
```

Comment:
Right now our we have the following in the elasticsearch config template:

```
network.host: <%= node['private_chef']['opscode-erchef']['vip']  %>
```

This likely needs to be changed to something ES specific.